api/api/models/media.py

```diff
@@ -88,7 +111,18 @@ class AbstractMedia(
         help_text="The top-level classification of this media file.",
     )
 
-    meta_data = models.JSONField(blank=True, null=True)
+    meta_data = models.JSONField(
+        blank=True,
+        null=True,
+        help_text=d(
+            """
```

Comment:
Consistency nit.

```suggestion
        help_text=d("""
```

Reply:
Ah, I've been using `black` to format files which was forcing this change (and it's valid under `ruff format` too). Swapped to using the latter locally and corrected this!